aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/events/LexV2Event.java

```diff
@@ -0,0 +1,228 @@
+        private String sentiment;
+        private SentimentScore sentimentScore;
+    }
+
+    @Data
+    @Builder(setterPrefix = "with")
+    @NoArgsConstructor
+    @AllArgsConstructor
+    public static class SentimentScore implements Serializable, Cloneable {
+        private double mixed;
+        private double negative;
+        private double neutral;
+        private double positiv;
+    }
+
+    @Data
+    @Builder(setterPrefix = "with")
+    @NoArgsConstructor
+    @AllArgsConstructor
+    public static class ProposedNextState implements Serializable, Cloneable {
+        private DialogAction dialogAction;
+        private Intent intent;
+    }
+
+    @Data
+    @Builder(setterPrefix = "with")
+    @NoArgsConstructor
+    @AllArgsConstructor
+    public static class DialogAction implements Serializable, Cloneable {
+        private String slotToElicit;
+        private String type;
+    }
+
+    @Data
+    @Builder(setterPrefix = "with")
+    @NoArgsConstructor
+    @AllArgsConstructor
+    public static class SessionState implements Serializable, Cloneable {
+        private ActiveContext[] activeContexts;
+        private Map<String, String> sessionAttributes;
+        private RuntimeHints runtimeHints;
+        private DialogAction dialogAction;
+        private Intent intent;
+    }
+
+    @Data
+    @Builder(setterPrefix = "with")
+    @NoArgsConstructor
+    @AllArgsConstructor
+    public static class ActiveContext implements Serializable, Cloneable {
+        private String name;
+        private Map<String, String> contextAttributes;
+        private TimeToLive timeToLive;
+    }
+
+    @Data
+    @Builder(setterPrefix = "with")
+    @NoArgsConstructor
+    @AllArgsConstructor
+    public static class TimeToLive implements Serializable, Cloneable {
+        private int timeToLiveInSeconds;
+        private int turnsToLive;
+    }
+
+    @Data
+    @Builder(setterPrefix = "with")
+    @NoArgsConstructor
+    @AllArgsConstructor
+    public static class RuntimeHints implements Serializable, Cloneable {
+        private Map<String, Map<String, Hint>> slotHints;
+    }
+
+    @Data
+    @Builder(setterPrefix = "with")
+    @NoArgsConstructor
+    @AllArgsConstructor
+    public static class Hint implements Serializable, Cloneable {
+        private RuntimeHintValue[] runtimeHintValues;
+    }
+
+    @Data
+    @Builder(setterPrefix = "with")
+    @NoArgsConstructor
+    @AllArgsConstructor
+    public static class RuntimeHintValue implements Serializable, Cloneable {
+        private String phrase;
+    }
+
+    @Data
+    @Builder(setterPrefix = "with")
+    @NoArgsConstructor
+    @AllArgsConstructor
+    public static class Transcription implements Serializable, Cloneable {
+        private String transcription;
+        private TranscriptionConfidence transcriptionConfidence;
+        private ResolvedContext resolvedContext;
+        private Map<String, Slot> ResolvedSlots;
```

Comment:
is this a typo or does it only work with Capital 'R'?

Reply:
Yes, must have missed that typo. Thanks for pointing it out.